src/utils/index.ts

```diff
@@ -46,11 +46,10 @@ export function isActiveItem(router: Router, href: string, singlePage?: boolean)
 }
 
 export function getLangPath(router: Router, lang: string) {
-    const path = router.hash ? normalizeHash(router.hash) : normalizePath(router.pathname);
+    const path = router.hash ? normalizeHash(router.hash) : router.pathname;
```

Comment:
normalizePath has two replaces

```path?.replace(/\.html$/, '')?.replace(/\/index$/, '/');```

the second one probably still needed

Reply:
The idea is to get the current path and replace only the language string